src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -119,16 +128,16 @@ private String getUsername(String userId) {
     return (String) currentUser.getProperty("username");
   }
 
-  // TODO: @salilnadkarni update checkStatus / updateStatus to use Datastore
-  private boolean checkStatus(String userId) {
-    if (!status.containsKey(userId)) {
-      status.put(userId, false);
-    }
-    return status.get(userId) == true;
+  private boolean checkGuessStatus(String userId, Entity currentRound) {
+    EmbeddedEntity userGuessStatuses = (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
+    boolean userGuessStatus = (Boolean) userGuessStatuses.getProperty(userId);
+    return userGuessStatus == true;
   }
 
-  private void updateStatus(String userId) {
-    status.replace(userId, true);
+  private void updateGuessStatus(String userId, Entity currentRound) {
```

Comment:
this also can be clearer. 

what is it doing? marking that this user guessed correctly in this round in this entity.

try to come up with something SO explicit it would be impossible to be confused.

heres a tip I like to use.

If its a private method, you have no bounds on making the name super long and explicit

if its public, keep it concise.